algebra-laws/js/src/main/scala/cats/algebra/laws/platform/Platform.scala

```diff
@@ -0,0 +1,10 @@
+package cats.algebra.laws.platform
+
+private[laws] object Platform {
```

Comment:
Can we use the same definition from `kernel-laws`? There, it is defined as `private[cats]` so it should be usable here.

Reply:
Yup, I think so. Adding the `kernel-laws` dependency also lets us drop a bunch of other dupes, like serialization laws.

Reply:
Done. Now also using Serialization, Eq, and (Partial)Order tests directly from kernel-laws.